tideways.c

```diff
@@ -3263,7 +3263,7 @@ void tideways_error_cb(int type, const char *error_filename, const uint error_li
 #else
 static void tideways_throw_exception_hook(zval *exception TSRMLS_DC)
 {
-	zval *exception_ce, *ex;
+	zend_class_entry *exception_ce, *ex;
```

Comment:
Good catch. I don't remember exception_ce and ex usage here, it is indeed unused. I will cleanup in another commit.